lib/msal-core/src/telemetry/TelemetryUtils.ts

```diff
@@ -1,15 +1,14 @@
-import { B2cAuthority } from "../authority/B2cAuthority";
-import { AADTrustedHostList } from "../utils/Constants";
 import { TENANT_PLACEHOLDER, EVENT_NAME_PREFIX } from "./TelemetryConstants";
 import { CryptoUtils } from "../utils/CryptoUtils";
 import { UrlUtils } from "../utils/UrlUtils";
+import { TrustedAuthority } from "../authority/TrustedAuthority";
 
 export const scrubTenantFromUri = (uri: string): String => {
 
     const url = UrlUtils.GetUrlComponents(uri);
 
     // validate trusted host
-    if (!AADTrustedHostList[url.HostNameAndPort.toLocaleLowerCase()]) {
+    if (!TrustedAuthority.IsInTrustedHostList(url.HostNameAndPort.toLocaleLowerCase())) {
```

Comment:
Since we no longer distinguish between AAD/B2C can we change this to just check if the authority is ADFS since ADFS does not have the concept of a tenant and this code will never be executed if the host is not trusted? It looks like the logic below will still work for B2C

Reply:
+1 to just checking for ADFS

Reply:
Done